libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -57,16 +55,14 @@ const AP_Param::GroupInfo AP_OAPathPlanner::var_info[] = {
     // @Path: AP_OADatabase.cpp
     AP_SUBGROUPINFO(_oadatabase, "DB_", 4, AP_OAPathPlanner, AP_OADatabase),
 
-#if APM_BUILD_TYPE(APM_BUILD_Rover)
-    // @Param: OPTIONS
+    // @Param{Rover}: OPTIONS
     // @DisplayName: Options while recovering from Object Avoidance
```

Comment:
It's an existing issue so no problem but at some point we should change the display name to start with "Object avoidance".  So perhaps it could be "Object avoidance recovery options"